### PR TITLE
Update back-up-data-before-switching-plans.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note to the document stating that "It's not required to back up data when you switch to another subscription that provides more and same services. User data will always be preserved and no action need be taken for any individual service that is present in both  expiring and replacement subscription assigned to the same user" as currently the users were under the assumption that it works but it needed clarity.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2673

File: microsoft-365/commerce/subscriptions/back-up-data-before-switching-plans.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ description: "Backup Outlook, OneDrive, Yammer, and SharePoint content before ch
 # Back up data before switching Microsoft 365 for business plans
 
 If a user will be switched to another subscription that has fewer data-related services or a user leaves the organization, a copy of their data that's stored in Microsoft 365 can be downloaded before they are switched to the new subscription.
+
+ > [!NOTE]
+ > It's not required to back up data when you switch to another subscription that provides more and same services. User data will always be preserved and no action need be taken for any individual service that is present in both  expiring and replacement subscription assigned to the same user.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/back-up-data-before-switching-plans.md]:
```suggestion
 > It's not required to back up data when you switch to another subscription that provides the same or more services. User data will always be preserved and no action needs to be taken for any individual service that is present in both the expiring and replacement subscriptions assigned to the same user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks.

File: microsoft-365/commerce/subscriptions/back-up-data-before-switching-plans.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ description: "Backup Outlook, OneDrive, Yammer, and SharePoint content before ch
 # Back up data before switching Microsoft 365 for business plans
 
 If a user will be switched to another subscription that has fewer data-related services or a user leaves the organization, a copy of their data that's stored in Microsoft 365 can be downloaded before they are switched to the new subscription.
+
+ > [!NOTE]
+ > It's not required to back up data when you switch to another subscription that provides the same or more services. User data will always be preserved and no action needs to be taken for any individual service that is present in both the expiring and replacement subscriptions assigned to the same user.
```
Comment (cmcatee-MSFT) [inline microsoft-365/commerce/subscriptions/back-up-data-before-switching-plans.md]: Please replace with the following:
`If you're moving a user to a subscription that has the same or more services, you don't need to back up user data. See [Move users to a different subscription](https://docs.microsoft.com/microsoft-365/commerce/subscriptions/move-users-different-subscription) .`

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please make the change as noted.

File: microsoft-365/commerce/subscriptions/back-up-data-before-switching-plans.md (hunk 2 above)
Comment (msbemba) [reply microsoft-365/commerce/subscriptions/back-up-data-before-switching-plans.md]: Thank you, @cmcatee-MSFT . I have made the changes.

**Pull request merged**